diff --git a/device/rockchip/rk2206/sdk_liteos/board/main.c b/device/rockchip/rk2206/sdk_liteos/board/main.c
--- a/device/rockchip/rk2206/sdk_liteos/board/main.c
+++ b/device/rockchip/rk2206/sdk_liteos/board/main.c
@@ -30,6 +30,7 @@
 #define  MAIN_TAG              "MAIN"
 int DeviceManagerStart();
 void IotInit(void);
+void lab01_lcd_example(void);
 
 /*****************************************************************************
  * Function    : main
@@ -51,6 +52,7 @@ LITE_OS_SEC_TEXT_INIT int Main(void)
         UartDebugInit();
         LosShellInit();
         shell_cmd_init();
+        lab01_lcd_example();
         //IotInit();
         // 开启驱动管理服务
         //DeviceManagerStart();
